docs/source/python/parquet.rst

```diff
@@ -244,6 +277,18 @@ an exception will be raised. This can be suppressed by passing
    pq.write_table(table, where, coerce_timestamps='ms',
                   allow_truncated_timestamps=True)
 
+Timestamps with nanoseconds can be stored without casting when using the
+more recent Parquet format version 2.0:
```

Comment:
@wesm now that `version="2.0"` and `data_page_version="2.0"` are two separate options, I think it is OK to actually properly document that you can use version 2.0 to store nanoseconds? (with of course the proper warning that this can hurt compatibility with other parquet readers, but it shouldn't produce invalid files anymore?)

Reply:
Yes, IMHO it is OK